pages/common/xzless.md

```diff
@@ -0,0 +1,9 @@
+# xzless
+
+> Filters and displays text from `.xz` and `.lzma` files.
+> Uses `less` to present the output.
+> More information: <https://manned.org/xzless.1>.
+
+- Display text:
```

Comment:
```suggestion
- Open a compressed text file:
```

Reply:
Can we say `Compressed text file`? Instead of that we could use "Display text from compressed file"?

Reply:
you're right, this works with binaries too
```suggestion
- Open a compressed file:
```